.github/workflows/build_wheels.yml

```diff
@@ -95,6 +115,50 @@ jobs:
         with:
           name: source_zip
           path: sdks/python/dist
+      - name: Clear dist
+        if: steps.is_rc.outputs.is_rc == 1
+        working-directory: ./sdks/python
+        run: |
+          rm -r ./dist
+          rm -rd apache-beam-source
+      - name: Add rc tag
```

Comment:
Let's call it RC Num to avoid conflation with RC tags.
How about `Rewrite SDK version to include RC number.`